solr/solr-ref-guide/src/query-settings-in-solrconfig.adoc

```diff
@@ -203,7 +203,7 @@ This setting controls whether search requests for which there is not a currently
 
 === maxWarmingSearchers
 
-This parameter sets the maximum number of searchers that may be warming up in the background at any given time. Exceeding this limit will raise an error. For read-only slaves, a value of two is reasonable. Masters should probably be set a little higher.
+This parameter sets the maximum number of searchers that may be warming up in the background at any given time. Exceeding this limit will raise an error. For read-only secondaries, a value of two is reasonable. Primarys should probably be set a little higher.
```

Comment:
Instead of "secondaries" and "primaries" (or "primarys"), lets call them "primary servers" and "secondary servers" in the reference guide.

Reply:
ok that's a good suggestion. Primarys was another typo. I'll add it.

Reply:
I've made this change @chatman